docker/server/Dockerfile.ubuntu

```diff
@@ -16,6 +16,7 @@ RUN sed -i "s|http://archive.ubuntu.com|${apt_archive}|g" /etc/apt/sources.list
         gnupg \
         locales \
         wget \
+        yasm \
```

Comment:
Why do we need it here?

Reply:
By default, yasm is not installed in ubuntu.  In libhdfs3 submodule, this PR (https://github.com/ClickHouse/libhdfs3/pull/22) introduced the file [src/common/crc_iscsi_v_pcl.asm]. 
That time they only updated the submodule and not the ClickHouse submodule reference.

When the ClickHouse's submodule has been updated to the latest commit, there was build error. To support the PR changes, the docker file has been updated to install yasm.

Reply:
Ok, the PR (https://github.com/ClickHouse/ClickHouse/pull/37949) raised already to support the changes but it has been merged. We used the PR as reference, as it already has been reviewed.

Reply:
It does not answer the question. Let's try to remove it and see what will happen - https://github.com/ClickHouse/ClickHouse/pull/46173